plugins/notifications-backend/src/service/types.ts

```diff
@@ -30,9 +30,16 @@ export type Notification = {
   actions: NotificationAction[];
 };
 
+// Keep in sync with FE: plugins/notifications-frontend/src/api/notificationsApi.ts
 export type NotificationsFilter = {
   containsText?: string;
   createdAfter?: Date;
   messageScope?: 'all' | 'user' | 'system';
   user?: string;
 };
+
+// Keep in sync with FE: plugins/notifications-frontend/src/api/notificationsApi.ts
+export type NotificationsQuerySorting = {
+  fieldName: 'title' | 'message' | 'created' | 'origin' | 'topic';
```

Comment:
I know that type script supports this 'value|value...' thing. I don't like it. why? cause it does not really enforce these values. it's not like enum in java, so we only get a headache from maintaining it when adding columns. I think it is better to just put string. we will validate it in getNotifications anyhow. if you have a way to make it work like enum then that's a bonus. Now I see we have FieldsAllowedToOrderBy. So I really don't understand why we need something other than string.

Reply:
There is little value in it for this case, indeed.
We do checking via `FieldsAllowedToOrderBy` but the later use of the value is very shallow so the benefit of having it typed is low. Removing it for backend.